projects/uitdatabank/docs/entry-api/project-flow.md

```diff
@@ -0,0 +1,60 @@
+
+It is important that you go through the checklist below in advance and validate your integration against it. By doing this in advance, you ensure a smooth transition to the production environment!
+
+### Checklist for a succesfull integration
+#### Mandatory fields
+- [ ] **name**: only use plain text 
+- [ ] **eventtype**: use a valid ID from the [UiTdatabank taxonomy](taxonomy.uitdatabank.be/terms)
+- [ ] **location**: UiTdatabank already contains a lot of locations. Use existing locations as much as possible. Therefore, a search on existing locations in UiTdatabank (see [docs](https://docs.publiq.be/docs/uitdatabank/55c6a99f43e7e-place-get)) should be built in in your application. Or alternatively (if all your events take place on the same location for example), the identifier of that location in UiTdatabank should be kept in your application so that each new event is created with the same location ID. Only when a location does not exist yet in UiTdatabank, a new one can be created. Always save the identifiers of the created locations in the application, so that later on you can update or delete the location if necessary
+- [ ] **calendar**: use the provided calendarTypes in a correct way, and use the ISO-8601 standard
+
+> In order to create an UiTPAS event, the fields `organizer` and `priceInfo` are also mandatory!
+
+#### Optional fields
+- [ ] **description**: only use plain text (and line breaks)
+- [ ] **age range**
+- [ ] **mediaObject (image)**: add an inspiring photo for your event, with respect to copyright
+- [ ] **organizer**: UiTdatabank already contains a lot of organizers. Use existing organizers as much as possible. Therefore, a search on existing organizers in UiTdatabank (see [docs](https://docs.publiq.be/docs/uitdatabank/c66fa72affb6e-organizer-get)) should be built in in your application. Or alternatively (if all your events have the same organizer for example), the identifier of that organizer in UiTdatabank should be kept in your application so that each new event is created with the same organizer ID. Only when an organizer does not exist yet in UiTdatabank, a new one can be created. Always save the identifiers of the created organizer in the application, so that later on you can update or delete the organizer if necessary.
+- [ ] **price information**: if applicable, add price information
+- [ ] **booking information**: if applicable, add booking (ticketing) information and a reservation period
+- [ ] **booking availability**: if applicable, inform about the event being sold out or fully booked
+- [ ] **status**: if applicable, inform about the event being cancelled or postponed
+- [ ] **contact information**: if applicable, add contact information
+- [ ] **labels**: if applicable, use labels to add extra information about the event. A label is at least 3 characters and maximum 50 characters long. Only text, digits and `-` are allowed
```

Comment:
This is not correct, technically any character aside from `;` is allowed and the minimum length is 2 characters and the max length is 255 characters. See for example https://docs.publiq.be/docs/uitdatabank/03d27d32fb489-labels-add. I would refrain from documenting such constraints in guides like this because as this proves it's often incorrect and even if we fix it here now, it usually becomes outdated quickly because no one thinks about updating it when the constraint changes.

Reply:
About the minimum length: I think it was 3 characters in the frontend (or backend) some time ago and we aligned the validation 1 or 2 years ago. The information I used is probably out of date. That proves your point about the downfall of refraining constraints like this :-)

Reply:
Yep that's very much likely something that has changed over time. I don't remember the details myself. Just that it's always hard to keep this in sync in all documentation 😄